pandas/tests/frame/test_duplicates.py

```diff
@@ -263,6 +263,13 @@ def test_drop_duplicates_tuple():
     tm.assert_frame_equal(result, expected)
 
 
+def test_drop_duplicates_empty():
```

Comment:
can you parameterize this on columns with ``[], ['A', 'B','C']`` as the values
IOW test the ``DataFrame(columns=[])`` and ``DataFrame(columns=['A', 'B', 'C'])``

Reply:
actually also need a test with no columns but an index e.g. ``DataFrame(index=[1, 2])``

so don't need to parameterize but test all of these cases (where the result is the input)